src/app/controllers/tournamentController.js

```diff
@@ -1,101 +1,70 @@
-      },
-      {
-        organization_id: 2,
-        name: "Drexel Gaming Association",
-        type: "School Club",
-        contact_email: "drexelgaming77@gmail.com",
-        website: "https://cms.cci.drexel.edu/dga/",
-      },
-    ],
-  };
-  //pool.query("SELECT * FROM Tournaments WHERE tournament_id")
   const id = parseInt(req.params.id);
-  let tournament = tournaments.data[id - 1];
-  let game_id = parseInt(tournament.game_id);
-  let game = games.data[game_id - 1];
-  let org_id = parseInt(tournament.organization_id);
-  let org = organizations.data[org_id - 1];
-  let start_date = new Date(tournament.start_date);
-  let end_date = new Date(tournament.end_date);
-  res.send(
-    `<!DOCTYPE html>
-         <html>
-         	<head>
-            	<title>${tournament.name}</title>
-            </head>
-            <body>
-            	<h1>${tournament.name}</h1>
-                <h3>Hosted by: ${org.name} - ${org.contact_email}</h3>
-                <h3>${start_date.toDateString()} - ${end_date.toDateString()}</h3>
-                <h3>Game: ${game.name}</h3>
-                <div>
-                	<h4>Description:</h4>
-                	<p>${tournament.description}</p>
-                </div>
-                <a>Home page</a>
-                <a>Registration</a>
-                <a>Brackets</a>
-            </body>
-         </html>
-    `,
-  );
+  try {
+    await pool
+      .query(
+        `SELECT 
+      Tournaments.name,
+      Games.name AS game_name,
+      Tournaments.start_date,
+      Tournaments.end_date,
+      Tournaments.description,
+      Organizations.name AS org_name,
+      Organizations.contact_email AS org_email
+      FROM Tournaments 
+      JOIN Games on Tournaments.game_id = Games.game_id
+      JOIN Organizations on Tournaments.organization_id = Organizations.organization_id
+      WHERE tournament_id=$1`,
+        [id],
+      )
+      .then((result) => {
+        console.log(result.rows);
+        let tournament = result.rows[0];
+        let name = tournament.name;
+        let game_name = tournament.game_name;
+        let org_name = tournament.org_name;
+        let org_email = tournament.org_email;
+        let start_date = new Date(tournament.start_date);
+        let end_date = new Date(tournament.end_date);
+        let desc = tournament.description;
+        return res.send(
+          `<!DOCTYPE html>
+           	<html>
+             	<head>
+                <title>${name}</title>
+              </head>
+              <body>
+                <h1>${name}</h1>
+                  <h3>Hosted by: ${org_name} - ${org_email}</h3>
+                  <h3>${start_date.toDateString()} - ${end_date.toDateString()}</h3>
+                  <h3>Game: ${game_name}</h3>
+                  <div>
+                    <h4>Description:</h4>
+                    <p>${desc}</p>
+                  </div>
+                  <p><a href=\"http://0.0.0.0:3000\">Home page</a></p>
```

Comment:
Home page link won't work being hardcoded like this, someone not running the server will hit their own interface rather than ours.